src/main/java/com/faforever/api/featuredmods/LegacyFeaturedModFileRepository.java

```diff
@@ -49,11 +58,18 @@ public LegacyFeaturedModFileRepository(EntityManager entityManager) {
         "    ON file.fileId = latest.fileId" +
         "       AND file.version = latest.version" +
         "  INNER JOIN updates_%1$s b" +
-        "    ON b.id = file.fileId;", innerModName, (version == null ? "" : "WHERE version <= :version")), FeaturedModFile.class);
+        "    ON b.id = file.fileId %3$s; ",
```

Comment:
`%3$s` ? What's that supposed to mean?

Reply:
3rd parameter for function `String.format`.
https://stackoverflow.com/questions/21329475/can-stringformatter-reuse-arguments